src/main/java/com/rapid7/client/dcerpc/mssamr/SecurityAccountManagerService.java

```diff
@@ -186,6 +187,11 @@ public SAMPRAliasGeneralInformation getAliasGeneralInformation(final AliasHandle
         return callExpectSuccess(request, "SamrQueryInformationAlias[1]").getAliasInformation();
     }
 
+    public List<SAMPRSIDInformation> getMembersInAlias(final AliasHandle aliasHandle) throws IOException {
```

Comment:
We should not expose SAMPRSIDInformation on the service layer since it is tied to the marshalling.

Reply:
👍

Reply:
@bli-r7 , others are same? It's all exposed at here 
https://github.com/sangdo-lee/smbj-rpc/blob/9d8311573d4a318b587ad67e514985c8b53c3bce/src/main/java/com/rapid7/client/dcerpc/mssamr/SecurityAccountManagerService.java#L292-L300,